client_src/js/printerManagementRunner.js

```diff
@@ -2873,50 +2876,61 @@ class dashUpdate {
           printerName = printer.printerName;
         }
         const printerCard = document.getElementById(
-          `printerCard-${printer._id}`,
+          `printerCard-${printer._id}`
         );
         if (printerCard) {
           const printName = document.getElementById(
-            `printerName-${printer._id}`,
+            `printerName-${printer._id}`
           );
           const printButton = document.getElementById(
-            `printerButton-${printer._id}`,
+            `printerButton-${printer._id}`
           );
           const webButton = document.getElementById(
-            `printerWeb-${printer._id}`,
+            `printerWeb-${printer._id}`
           );
           const hostBadge = document.getElementById(`hostBadge-${printer._id}`);
           const printerBadge = document.getElementById(
-            `printerBadge-${printer._id}`,
+            `printerBadge-${printer._id}`
           );
           const socketBadge = document.getElementById(
-            `webSocketIcon-${printer._id}`,
+            `webSocketIcon-${printer._id}`
           );
           const printerPrinterInformation = document.getElementById(
-            `printerPrinterInformation-${printer._id}`,
+            `printerPrinterInformation-${printer._id}`
           );
           const printerOctoPrintInformation = document.getElementById(
-            `printerOctoPrintInformation-${printer._id}`,
+            `printerOctoPrintInformation-${printer._id}`
           );
           const printerSortIndex = document.getElementById(
-            `printerSortIndex-${printer._id}`,
+            `printerSortIndex-${printer._id}`
           );
           const printerGroup = document.getElementById(
-            `printerGroup-${printer._id}`,
+            `printerGroup-${printer._id}`
           );
 
-          printerGroup.innerHTML = printer.groups.map(g => g.name).join();
+          printerGroup.innerHTML = printer.groups.map((g) => g.name).join();
 
           if (typeof printer.octoPrintSystemInfo !== "undefined") {
             if (
               typeof printer.currentProfile !== "undefined" &&
               printer.currentProfile !== null
             ) {
-              printerPrinterInformation.innerHTML = `
+              if (
+                typeof printer.octoPrintSystemInfo["printer.firmware"] ===
+                "undefined"
+              ) {
+                printerPrinterInformation.innerHTML = `
+             <small title="Please connect and resync to display printer firmware"><b>Firmware: </b>Unknown</small><br>
```

Comment:
Does the suggested action really solve the issue of firmware not arriving?
I'd expect the connection to be wrong (like websocket not correctly setup), as was my own experience earlier.

So the solution would be 'the connection has not been setup succesfully, so cant acquire firmware status'